### PR TITLE
Assign1

### PR DESCRIPTION
**Pull request by sanderszzw**
reviewers: @mikwat @jpang-h4 @duaneatat

### CODE REVIEW

File: assign1/assign1-sol/app.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    $scope.lunch = {};
+    $scope.lunch.inputString = '';
+    $scope.lunch.notification = '';
+    $scope.lunch.inputType = '';
+
+    // function to perform item check. If input is empty string, promp user
+    // to give the input. If the number of input items is larger or equal
+    // to 3, show 'Enjoy!', otherwise show 'Too much!'.
+    $scope.performCheck = function() {
+      if (!$scope.inputString) {
```
Comment (mikwat) [inline assign1/assign1-sol/app.js]: Should this be `$scope.lunch.inputString`?

File: assign1/assign1-sol/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </style>
+  </head>
+<body>
+   <div class="container">
+     <h1>Lunch Checker</h1>
+
+         <div class="form-group">
+             <input id="lunch-menu" type="text"
+             placeholder="list comma separated dishes you usually have for lunch"
+             class="form-control" ng-model="inputString" ng-class="lunch.inputType">
```
Comment (mikwat) [inline assign1/assign1-sol/index.html]: And this `lunch.inputString`? It's always good practice to have a dot in your `ng-model`:
http://stackoverflow.com/questions/17606936/why-dont-the-angularjs-docs-use-a-dot-in-the-model-directive

**Review by mikwat — APPROVED** [review]
Did you push this to github.io so that it can be demo'ed?

**Comment by sanderszzw** [comment]
@mikwat Thank you for the review. Do you mean this io page? https://sanderszzw.github.io/coursera-web/assign1/assign1-sol/

**Comment by mikwat** [comment]
👍 Great, looks good! Thanks @sanderszzw

**Pull request merged**